### ISSUE TITLE
No pretrained model? + RGBA Warning, Odd generation after a few epochs

### ISSUE THREAD

**Issue by JordanMicahBennett** [ConorLazarou/PokeGAN#3]
Question:
What version of python did you use? (I am on Py36)
Any chance that you have the saved model somewhere?

Potential problem:
I may be mistaken, but I had expected to see transparent regions around early figures generated even after only 2 epochs.

Instead , I see a warning "Palette images with Transparency expressed in bytes should be converted to RGBA images", and I also see images being returned as the item below, with colour where I anticipated transparent regions to me.

![image](https://user-images.githubusercontent.com/3666405/111657962-ebe15c00-87d9-11eb-9707-75ba26c5f343.png)

Note that the plot is also the same:

![image](https://user-images.githubusercontent.com/3666405/111658161-16cbb000-87da-11eb-980a-27312eb8148f.png)

In case you don't have time to answer, my plan is to dig down into the offending warning PIL/Image file, and adjust the Image loading function somehow to perform adequate rgba conversion. These issues may be arising from using invalid version of python, wrt main codes.

//////////////////////////////
Edit: After some more epochs, things start to seem okay. I would still like the trained model though, for my little computer has gone through a bit :)

![image](https://user-images.githubusercontent.com/3666405/111662104-a1fa7500-87dd-11eb-8eb4-17d14111a70d.png)

**Comment by ConorLazarou** [comment]
Hey Jordan, thanks for the interest! I used python 3.7, but everything should be python 3.6 compatible.

This GAN was set up to use RGB images, not RGBA. I did some pre-processing of the sprites to remove the alpha layer and set the background to be white where the transparency used to be 1 (those regions are black in most of the sprites). Whether or not this was a good idea, who knows. To fix it, you should either do the same pre-processing that I did or you'll need to change the architecture to handle 4 channel input and output instead of 3 channel.

As for seeing all blobs at first, that's because of the [ColourPicker](https://towardsdatascience.com/i-generated-thousands-of-new-pokemon-using-ai-f8f09dc6477e) technique; it takes a bit longer for the GAN to learn that one of the palette colours should be white and that that colour should be used at the outside.

**Comment by JordanMicahBennett** [comment]
> Hey Jordan, thanks for the interest! I used python 3.7, but everything should be python 3.6 compatible.
> 
> This GAN was set up to use RGB images, not RGBA. I did some pre-processing of the sprites to remove the alpha layer and set the background to be white where the transparency used to be 1 (those regions are black in most of the sprites). Whether or not this was a good idea, who knows. To fix it, you should either do the same pre-processing that I did or you'll need to change the architecture to handle 4 channel input and output instead of 3 channel.
> 
> As for seeing all blobs at first, that's because of the [ColourPicker](https://towardsdatascience.com/i-generated-thousands-of-new-pokemon-using-ai-f8f09dc6477e) technique; it takes a bit longer for the GAN to learn that one of the palette colours should be white and that that colour should be used at the outside.

Thanks for the quick answer Lazarou. Any chance that you have the pre-trained model somewhere?

Epoch 199:\ in my machine:
![image](https://user-images.githubusercontent.com/3666405/111708518-ff111d80-8813-11eb-97fb-89cf5e601174.png)

Within 200 epochs, I saw proper results forming. 

However, got a crash after that time.

![image](https://user-images.githubusercontent.com/3666405/111708738-68912c00-8814-11eb-8dc9-123f410bfb69.png)

Wondering if that crash happened because I tried to invoke gpu to do training on a separate task  which uses tensorflow/gpu compute on my machine.

**Comment by JordanMicahBennett** [comment]
I may be missing something, but I have been trying to load the saved checkpoint at epoch 199 or 100 before the crash above. I see various ways, I can't seem to find loading strategy for loading the models saved for the generator.

Gain, thanks for your marvelous code

**Comment by JordanMicahBennett** [comment]
Thanks for the quick answer Lazarou. Marvelous code. Any chance that you have the pre-trained model somewhere?

Epoch 199 on py36 on my machine:
![image](https://user-images.githubusercontent.com/3666405/111708518-ff111d80-8813-11eb-97fb-89cf5e601174.png)

Within 200 epochs, I saw proper results forming. 

However, got a crash after that time.

![image](https://user-images.githubusercontent.com/3666405/111708738-68912c00-8814-11eb-8dc9-123f410bfb69.png)

Wondering if that crash happened because I tried to invoke gpu to do training on a separate task  which uses tensorflow/gpu compute on my machine.
